### ISSUE TITLE
Double pageview generated with single Matomo Pageview tag

### ISSUE THREAD

**Issue by gonerator** [matomo-org/tag-manager#66]
Hi! Tag Manager is great, but I am having an issue while setting it up.

I created container, included JS into my website, and then created Matomo pageview tag to trigger on each pageview (I suppose this is the correct way):

![image](https://user-images.githubusercontent.com/16865753/44602396-cdf91980-a7df-11e8-971a-126c4dd0039d.png)

This is the trigger:

![image](https://user-images.githubusercontent.com/16865753/44602511-1c0e1d00-a7e0-11e8-8ff0-776676a131a0.png)
 
And when loading the website I have two pageview requests generated:

![image](https://user-images.githubusercontent.com/16865753/44602599-58da1400-a7e0-11e8-8e35-ba719bf8b7a0.png)

The Debugger:

![image](https://user-images.githubusercontent.com/16865753/44602627-698a8a00-a7e0-11e8-978c-eeb9b66a432b.png)

Can anyone help to tell on the reason for this, or maybe it's little bug?

As a test I included standard tracking code for Analytics as Custom HTML tag with the same trigger, and then I have single pageview tracker for each load of the page, as it should be.

Thanks in advance for the help!

**Comment by tsteur** [comment]
Any chance you maybe still have the regular Matomo tracking code embedded?

**Comment by gonerator** [comment]
@tsteur thanks for the answer! I'm sure there is no "old" code, as this is a fresh website and I started off with just MTM code. When I remove this tag, no requests are sent when page is loaded.

I have a 50% scroll tag that sends Matomo Event, and when this is triggered, it's sent properly, so there must be tracker initiated in the first place I suppose?

But as far as I checked Matomo Tag Manager container is not performing Analytics tracker on it's own? That's why I should have pageview tag triggered on all pageviews? If yes, then that's exactly what I did.
